setup/db/db/schema-490to491.sql

```diff
@@ -23,3 +23,11 @@
 UPDATE `cloud`.`roles` SET `description`='Default user role' WHERE `id`=4 AND `role_type`='User' AND `description`='Default Root Admin role';
 
 
+ALTER TABLE cloud.load_balancer_cert_map ENGINE=INNODB;
+ALTER TABLE cloud.monitoring_services ENGINE=INNODB;
+ALTER TABLE cloud.nic_ip_alias ENGINE=INNODB;
+ALTER TABLE cloud.sslcerts ENGINE=INNODB;
+ALTER TABLE cloud.op_lock ENGINE=INNODB;
+ALTER TABLE cloud.op_nwgrp_work ENGINE=INNODB;
```

Comment:
For op_lock and op_nwgrp_work tables engine is MEMORY, not sure of the reason. Was there any perf. issues addressed? These exists since the start.